packages/@azure/servicebus/data-plane/test/topicFilters.spec.ts

```diff
@@ -168,7 +168,7 @@ async function addRules(
   }
 }
 
-describe("Topic Filters -  Add Rule - Positive Test Cases", function(): void {
```

Comment:
Right now, for us, we can point the failed tests clearly without "Topic Filters"(in the description).
For future, I recommend keeping this. Any other thoughts ?

Reply:
Can you elaborate Harsha?

My intention was that anyone familiar with the Service Bus project should be able to figure out what is test is failing looking at the output.

The output for failed tests in CI will be whats in the `describe` + whats in the `it`
The output locally for failed tests is just the `it`

In this case "Add Rule" in the describe should be enough hint that we are dealing with Topic Filters
"Add True Filter" in the `it` should be enough that we are testing true filters.

What would be the advantage of keeping "Topic Filters" in the name?

Reply:
Updated in https://github.com/Azure/azure-sdk-for-js/pull/1237/commits/8bffc29c0908a2587a4326a66563c699970615a9